Update integration variable limit increase (#5385)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md

     `*` This section is optional, but if you specify a variable, both `Label` and `Name` properties are required.

1. Click **+ Add another** to add another variable. You can add up to eight variables.

>    > **Note:**  The apps migrated from OIN Manager with more than eight variables can retain all existing variables but can't add new ones. However, you can update or delete the existing variables.

Reviewer: ```suggestion
   > **Note:**  Apps that are migrated from the OIN Manager and that have more than eight variables can retain those variables, but you can't add new ones. However, you can update or delete the existing variables.
```
